docs/geedocs/5.2.3/answer/7160003.html

```diff
@@ -412,6 +412,11 @@ <h5>
           <td>Geserver fails to startup fully due to conflicting protobuf library</td>
           <td>Run <code>pip uninstall protobuf</code> to uninstall the protobuf library installed by pip. </td>
         </tr>
+	<tr>
```

Comment:
this fix need to be added to list of `Resolved Issues` instead of list of `Known Issues`. Also when adding it there ensure that it listed in order as other fixed issues. In this case, you want to add after fixed issue 225.